### PR TITLE
JSON data reading

### PR DESCRIPTION
**Pull request by pfaraee**
-Program now reads JSON files which will be used to save task data.

### CODE REVIEW

**Comment by moisesbaez** [comment]
There is an error in the code on line 36 of TodoActivity.java:

textView = (TextView)findViewById(R.id.textView);

There is no TextView on the XML layout. What happened to it? Make sure your code works before submitting a pull request. In this case, test out the JSONSerializer to ensure that it is saving a task, even if you're just faking it right now.

**Comment by pfaraee** [comment]
Oh shit yeah I deleted it because I was gonna get the layout from Brady but
it works if you run it. My bad

**Comment by moisesbaez** [comment]
I think next time just keep the code as is, let whoever is going to edit it make the changes. You might also want to make a note after you make the pull request, or create a task on the Scrum board. Leaving it as is allows us to compile and run the code and see the results. This can be beneficial to whoever requires the code to understand what is going on.

Clean up the language too. Everyone, that means the whole internet, can see this cause it's public lol. Keep it professional :smile:

File: app/src/main/java/android/supremeaa/todo/Controller/TaskSerializer.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        task.setDate(date);
+        int priority = jsonObject.optInt("priority", task.getPriority());
+        task.setPriority(priority);
+
+        return true;
+    }
+    public static String loadJSONFromAsset(Context c) {
+        String json = null;
+        try {
+            InputStream is = c.getAssets().open("taskdata.json");
```
Comment (moisesbaez) [inline app/src/main/java/android/supremeaa/todo/Controller/TaskSerializer.java]: Change "taskdata.json" to "JSON/taskdata.json"

**Comment by moisesbaez** [comment]
Line 73 of TaskSerializer.java causes an issue. It can't read that json file. Make sure the path is correct. It should read:

 "JSON/taskdata.json"

**Comment by pfaraee** [comment]
Now reads JSON arrays and displays them in the listview.

**Pull request merged**